libs/auth/src/artifact_authenticator.ts

```diff
@@ -35,10 +35,20 @@ interface ArtifactSignatureBundle {
 }
 
 /**
- * An artifact authenticator that uses digital signatures to verify the authenticity of
+ * The API for an artifact authenticator that uses digital signatures to verify the authenticity of
  * machine-exported artifacts, meeting VVSG2 data protection requirements
  */
-export class ArtifactAuthenticator {
+export interface ArtifactAuthenticatorApi {
```

Comment:
Following a pattern similar to what I landed on for the auth classes, e.g. `DippedSmartCardAuth` + `DippedSmartCardAuthApi`, where the `*Api` interface makes mocking in application-level tests easier